sql/updates/world/6.x/2016_09_26_00_world.sql

```diff
@@ -0,0 +1,3 @@
+-- Changing subzone 6170 (Northshire, Elwynn). Was zone 12 before, not right.
+-- Part of my total fix for graveyard problems (https://github.com/TrinityCore/TrinityCore/issues/14089)
+UPDATE graveyard_zone SET GhostZone=6170 WHERE ID=105 AND Faction=469
```

Comment:
your SQL:
```sql
UPDATE graveyard_zone SET GhostZone=6170 WHERE ID=105 AND Faction=469
```
Recommended SQL:
```sql
UPDATE `graveyard_zone` SET `GhostZone`= 6170 WHERE `ID`= 105 AND `Faction`= 469;
```
Also please add a CarriageReturn/LineFeed at the end of the last text line.
The writing cursor should always rest on a new blank line below all text files.